sdk/monitor/monitor-opentelemetry-exporter/src/export/trace.ts

```diff
@@ -72,7 +72,7 @@ export class AzureMonitorTraceExporter implements SpanExporter {
             error: new Error("Failed to persist envelope in disk."),
           };
     } catch (ex: any) {
-      return { code: ExportResultCode.FAILED, error: ex };
+      return { code: ExportResultCode.FAILED, error: ex }; // eslint-disable-line @typescript-eslint/no-unsafe-assignment
```

Comment:
May I suggest updating the type of `ex` to `unknown` in the catch clause and casting it to the `error` property type at the assignment time?

Reply:
Leaving as `any` but doing `ex as Error` which fixes the lint issue